features/anon-user.feature

```diff
@@ -13,3 +13,21 @@ Feature: Anonymous user
     # Because Behat users won't have CAS sessions...
     Then I should see the heading "Application Not Authorized to Use CAS"
     And print current URL
+
+  @api @javascript @search_page
+  Scenario: Verify Anonymous users can see and use search page
+    Given I am an anonymous user
+    When I go to the homepage
+    When I click the element ".search-button"
+    Given for "q" I enter "test"
+    Then I submit the "[name='gs']" form
+    Then the url should match "^\/\#gsc\.tab\=0\&gsc\.q=test.*?$"
+    And I should see the heading "Search"
+    Then I wait for 5 seconds
```

Comment:
@juanmitriatti I think this ticket is meant to test the search functionality on the 404 page itself. I have added a comment on the Jira ticket itself to help clarify.

Reply:
Do you think that we have to drop the current test? Or should we keep this and add a new step for the 404 page search

Reply:
@juanmitriatti The current tests are good to have. You can just add another.